src/javascript/api-data.js

```diff
@@ -0,0 +1,21 @@
+async function getTemp(newCity = 'Toronto') {
+  const weatherData = await fetch(`https://api.openweathermap.org/data/2.5/weather?q=${newCity}&units=metric&appid=550a45bc4aa9613e4687926216e69765`)
+    .then((response) => response.json()).then((data) => data);
```

Comment:
**REQUIRED**
Let's add error handling to handle the case when an error is thrown while processing requests. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/weather_app/error_handling_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59622133/))